x-pack/plugins/observability_solution/slo/public/embeddable/slo/alerts/handle_explicit_input.tsx

```diff
@@ -47,7 +47,7 @@ export async function resolveEmbeddableSloUserInput(
               />
             </QueryClientProvider>
           </KibanaContextProvider>,
-          { i18n: coreStart.i18n, theme: coreStart.theme }
+          coreStart
```

Comment:
Using more of the full coreStart object allows the `analytics` dependency (which is currently "optional") to be easily provided.